docs/adr/005-use-django-for-back-end-web-framework.md

```diff
@@ -0,0 +1,45 @@
+# 5. Back-end web framework
+
+Date: 2021-07-02
+
+## Status
+
+Proposed
+
+## Context
+
+We discussed different options for our tech stack on GitHub as a team — see discussion in [#25](https://github.com/18F/OPRE-Unicorn/issues/25). Afterwards, the engineering team met to discuss back-end tech stack options in particular. 
+
+### Explanation of the tech concept
+
+Django, Flask, Rails, Sinatra -- these are all examples of web frameworks. Web frameworks are pre-written code that give developers tools and features right away when they start new web application projects. 
+
+Using a web framework, we won't be starting off writing code on a totally blank page. We'll start with some of the tools and features we need already in place, such as code to fetch and send data to and from a database. That will help us jump more quickly to writing project-specific code. 
+
+### Resources for further reading
+
++ https://www.djangoproject.com/
++ https://docs.djangoproject.com/en/dev/faq/general/#faq-general 
++ Django Admin: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Admin_site
+
+## Decision
+
+We found concensus on using Django, although we considered Flask as well. We felt that the features Django offers out-of-the-box -- such as tools to interact with the database layer, and the Django Admin dashboard -- are ones the project is likely to use and benefit from. 
```

Comment:
There was a lot of helpful content in the comments thread on Issue #25, some of which would be helpful to include here to provide more context about what these features are.  For example: 

> Features that can be used out of the box for this application without too much development work needed:
> 
> - ORM: since there is a need for a database, each Django project will come with the configuration to a database
> - Authentication/Authorization: There will be some form of authentication/authorization needed, Django already comes with this package, and more if we need to use more specific authentication
> - Admin interface: This will be good for administrator to edit data easily

Also the maturity of Django, TTS's familiarity/success with it, etc.

Reply:
Gotcha! I included a link to the Issue in the ADR, so readers can always click through and read more. I can copy over some of those bullets into the ADR if that would be helpful. As a directional note, I like to think about ADRs as lightweight journal entires — see the [example ADR in the recent 18F blog post](https://18f.gsa.gov/2021/07/06/architecture_decision_records_helpful_now_invaluable_later/#whats-an-adr) for en example of the length & depth of a typical ADR.